test/services/dumpers/agent-nodejs/expected/mariadb/.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+npm-debug.log
+.env
```

Comment:
No extra carriage at the end of our files ?

Reply:
Will ad them 🤷

Reply:
👍